stream-chat-android-ui-common/src/main/res/values/attrs_avatar_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+
+    Licensed under the Stream License;
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<resources>
+    <bool name="stream_ui_excludeCurrentUserFromChannelAvatars">true</bool>
+</resources>
```

Comment:
Well, it's not defining any attrs. I would call it either `config.xml`, `bools.xml` or `config_avatar_view.xml`. Also, it's in the common module, are we going to use it in Compose as well?

Reply:
Renamed to config